lib/redis_client/decorator.rb

```diff
@@ -47,8 +47,8 @@ def with(*args)
       end
       ruby2_keywords :with if respond_to?(:ruby2_keywords, true)
 
-      def pipelined
-        @client.pipelined { |p| yield @_pipeline_class.new(p) }
+      def pipelined(**kwargs)
+        @client.pipelined(**kwargs) { |p| yield @_pipeline_class.new(p) }
```

Comment:
```suggestion
      def pipelined(exception: false)
        @client.pipelined([**kwargs](exception: false)) { |p| yield @_pipeline_class.new(p) }
```

It's micro-optimization territory, but I'd rather avoid allocating a Hash on every call here.

We control both methods, so we don't need blind delegation for forward compatibility.

Reply:
Got it, however I think you meant the following (with default to `true`)
```ruby
def pipelined(exception: true)
  @client.pipelined(exception: exception) { |p| yield @_pipeline_class.new(p) }
end 
```

Reply:
I did yes.